.github/scripts/setup_test_profiling_env.sh

```diff
@@ -25,18 +25,26 @@ if [ "$#" -ne 5 ]; then
     echo "usage: $0 <jdk_version> <run_id> <run_number> <run_attempt> <module>"
 fi
 
-if [[ "$1" == "17" ]];
+if [[ "$1" -ge "17" ]];
 then
   curl https://static.imply.io/cp/$JAR_INPUT_FILE -s -o $JAR_OUTPUT_FILE
 
+  # Run 'java -version' and capture the output
+  output=$(java -version 2>&1)
+
+  # Extract the version number using grep and awk
+  jvm_version=$(echo "$output" | grep "version" | awk -F '"' '{print $2}')
```

Comment:
depending on the JVM implementation this might not give us what we want.
I'd recommend using the code that's already been tested here 

https://github.com/apache/druid/blob/c324e377512bfa815e0e71fc3dbdb69934d9951e/examples/bin/run-java#L27